discord-stubs/message.pyi

```diff
@@ -69,7 +70,7 @@ class Message:
     call: Optional[CallMessage]
     reference: Optional[MessageReference]
     mention_everyone: bool
-    mentions: List[Union[User, Member]]
+    mentions: List[Union[User, Member, Client.user]]
```

Comment:
This should be `ClientUser` from `.user`

Reply:
Ok, I'm going to fix that in the next commit